client/client.go

```diff
@@ -303,8 +303,9 @@ func (r *NotaryRepository) RemoveTarget(targetName string) error {
 	return nil
 }
 
-// ListTargets lists all targets for the current repository
-func (r *NotaryRepository) ListTargets() ([]*Target, error) {
```

Comment:
I believe it's because you keep commenting on the deleted code rather than the added code.